shared/cfg/guest-hw.cfg

```diff
@@ -163,7 +163,6 @@ variants:
         # available, it will also try to get first available pf.
         device_name = eth1
         nettype = bridge
-        pf_filter_re = "82576 Gigabit Network"
```

Comment:
It is because we define it in other cfg for different pfs.  You can refer to https://github.com/autotest/tp-qemu/pull/1026.